src/home/fetch-github/fetch-and-display-previews.ts

```diff
@@ -66,6 +66,10 @@ export async function fetchAvatars() {
   const urlPattern = /https:\/\/github\.com\/(?<org>[^/]+)\/(?<repo>[^/]+)\/issues\/(?<issue_number>\d+)/;
 
   const avatarPromises = cachedTasks.map(async (task) => {
+    if (!task.preview.body) {
+      return Promise.resolve();
+    }
```

Comment:
What exactly are the consequences of this? It's a bad idea to make unnecessary changes in the pulls.